pandas/_libs/tslib.pyx

```diff
@@ -830,24 +829,6 @@ class Timestamp(_Timestamp):
 # ----------------------------------------------------------------------
 
 
-cdef inline bint _check_all_nulls(object val):
-    """ utility to check if a value is any type of null """
```

Comment:
is this used anywhere?

Reply:
Yes, its used once here in _libs.missing.